src/pages/Dashboard/IOTokenDistributionPanel.tsx

```diff
@@ -0,0 +1,154 @@
+import { AoTokenSupplyData, mIOToken } from '@ar.io/sdk';
+import Placeholder from '@src/components/Placeholder';
+import useTokenSupply from '@src/hooks/useTokenSupply';
+import { useGlobalState } from '@src/store';
+import { formatWithCommas } from '@src/utils';
+import { useEffect, useState } from 'react';
+import { Cell, Pie, PieChart, ResponsiveContainer, Text } from 'recharts';
+
+const TOTAL_IO = 1_000_000_000;
```

Comment:
pretty sure this is exported from the sdk (and should probably use that)

Reply:
Now using getTokenSupply().total value if available and falling back to the hardcoded value here as a just in case.